controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -1516,10 +1513,160 @@ bool master_thread::handle_intel_slave_join(Socket *sd, ieee1905_1::CmduMessageR
     return true;
 }
 
-bool master_thread::handle_non_intel_slave_join(Socket *sd,
-                                                std::shared_ptr<ieee1905_1::tlvWscM1> tlvwscM1,
-                                                std::string bridge_mac, std::string radio_mac,
-                                                ieee1905_1::CmduMessageTx &cmdu_tx)
+static inline std::vector<uint8_t> operating_class_to_channel_set(uint8_t operating_class)
```

Comment:
Let's assume we would have wanted this function. It certainly wouldn't be here. it will be on son_wireless_utils.
Second, operating class has also bw which you ignored.

Reply:
You are right, I will move it to son_wireless_utils. What do you mean by "let's assume we would want this function"? Its not matter of wanting it, I needed it for getting the right channel and understanding weather it is a 5g or 2.4g radio.

Let me remind you that this is still draft, I didn't explain why I did everything I did in this commit so try to keep an open mind:)

I ignored everything other than channel on purpose - all we need at the moment is the channel.